pymc3/distributions/continuous.py

```diff
@@ -309,35 +633,92 @@ class Beta(UnitContinuous):
     """
     def __init__(self, alpha=None, beta=None, mu=None, sd=None, *args, **kwargs):
         super(Beta, self).__init__(*args, **kwargs)
-        alpha, beta = self.get_alpha_beta(alpha, beta, mu, sd)
+        alpha, beta = self.parametrize(alpha, beta, mu, sd)
         self.alpha = alpha
         self.beta = beta
         self.mean = alpha / (alpha + beta)
         self.variance = alpha * beta / (
-            (alpha + beta) ** 2 * (alpha + beta + 1))
-
-    def get_alpha_beta(self, alpha=None, beta=None, mu=None, sd=None):
+            (alpha + beta) ** 2 * (alpha + beta + 1))        
+    
+    def parametrize(self, alpha=None, beta=None, mu=None, sd=None):
         if (alpha is not None) and (beta is not None):
             pass
+            #mu = alpha / (alpha + beta)
+            #sd = (alpha * beta / (alpha + beta) ** 2 / (alpha + beta + 1)) ** 0.5
         elif (mu is not None) and (sd is not None):
-            alpha = mu * sd
-            beta = (1 - mu) * sd
+            alpha = (1 - mu) * (mu ** 2) / (sd ** 2) - mu
```

Comment:
The previous transform (`alpha = mu * sd` and `beta = (1 - mu) * sd`) might actually not be wrong. But I interpreted the `mu` and `sd` parameters to be the mean and square-root of the variance, respectively, so just derived the alternative transform by substitution.

If my interpretation is correct then the previous transform was wrong. Otherwise it is me that is wrong.